charts/uptime-kuma/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: uptime-kuma
 sources:
   - https://github.com/louislam/uptime-kuma
 type: application
-version: 2.7.10
+version: 2.7.0
```

Comment:
needs a bump to `2.7.11` or something higher than `2.7.0`.

Reply:
Good catch. 2.8 is what I meant xD

```suggestion
version: 2.8.0
```